image/demos/style.css

```diff
@@ -0,0 +1,71 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 20px;
+  font: 300 16px/1.5 "Roboto", Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+  color: #111;
+  background-color: #fdfdfd;
+  -webkit-font-smoothing: antialiased;
+  width: 100%;
+  max-width: 1000px;
+}
+
+section {
+  background: #F2F2F2;
+  padding: 14px;
+  margin-bottom: 48px;
+}
+
+a {
+  color: #2a7ae2;
+  text-decoration: none;
+}
+a:visited {
+  color: #1756a9;
+}
+a:hover {
+  color: #111;
+  text-decoration: none;
+}
+
+button, .button {
+  font-family: "Roboto", Helvetica,Arial,sans-serif,"Apple Color Emoji","Segoe UI Emoji","Segoe UI Symbol";
+  text-transform: uppercase;
+  font-weight: 700;
+  font-size: 16px;
+  padding: 4px 14px;
+  margin-right: 8px;
+  border: 3px solid #222;
+  background: transparent;
+  outline: none;
+  color: #222;
+  background: white;
+  cursor: pointer;
+}
+button:hover {
+  background: #6FC9C6;
+  color: white;
+}
+button[disabled] {
+  opacity: 0.5;
+}
+
+.image {
+  margin-right: 10px;
+  outline: 3px solid #222;
+}
+
+.player-container {
```

Comment:
heh, you probably don't need player-container and visualizer-container. they are very music specific 😅

Reply:
Oops, my bad! Done.